docs/.vuepress/plugins/dump-docs-data/canonicals.js

```diff
@@ -25,6 +25,18 @@ async function generateCommonCanonicalURLs(currentCanonicals) {
     headers: fetchCommonHeaders
   });
   const docsData = await response.json();
+
+  // TODO: Mocked.
+  docsData.patches = [
```

Comment:
This mock causes happy tests on localhost. But on production/staging, there would be some problems. Can you remove that mock?

Reply:
Removed within https://github.com/handsontable/handsontable/pull/10079/commits/e539c2a3a296ae4735d921341bed747ab79092d7 and https://github.com/handsontable/handsontable/pull/10079/commits/1f924816080aa8ce52bd36efb36c252675e92e20.